src/change_stream.ts

```diff
@@ -416,6 +462,20 @@ export class ChangeStreamCursor extends Cursor<AggregateOperation, ChangeStreamC
     }
   }
 
+  close(): void;
+  close(callback: Callback): void;
+  close(options: CursorCloseOptions): Promise<void>;
+  close(options: CursorCloseOptions, callback: Callback): void;
+  close(
+    options?: CursorCloseOptions | Callback<void>,
+    callback?: Callback<void>
+  ): Promise<void> | void {
+    if (typeof options === 'function') (callback = options), (options = {});
+    options = options || {};
+    this.emit(Cursor.CLOSE);
+    return super.close(options, callback as Callback);
```

Comment:
I think no need to cast here?

Reply:
@emadum ping

Reply:
I was getting a TS error without casting, but this entire change is now unnecessary, it was based on a misunderstanding I had.